src/app/component/auth/sign-in-new/sign-in-new.component.scss

```diff
@@ -0,0 +1,205 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
+
+$fontOpenSans : OpenSans, sans-serif;
```

Comment:
Please rename without font name. It can be $title-font and so on.

Reply:
Fixed